### PR TITLE
Add MediaLink v2.66

### PR DESCRIPTION
**Pull request by bashu**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/medialink.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'medialink' do
+  version '2.66'
+  sha256 '91c18bf24b546476397ca512e0418e152262ba7dd6362c671328f7a8e7199e38'
```
Comment (miccal) [inline Casks/medialink.rb]: The `version` should be `:latest` and the `sha256` should be `:no_check` since the download URL is unversioned.

Comment (bashu) [inline Casks/medialink.rb]: @miccal fixed

**Comment by bashu** [comment]
@miccal please review

**Comment by Numbermaniac** [comment]
> !!! Casks added outside Casks directory: Casks/medialink.rb

Not sure why Travis is throwing this...it looks like you added it in the right place.

**Comment by jawshooah** [comment]
> Not sure why Travis is throwing this...it looks like you added it in the right place.

Yup, this was fixed in https://github.com/caskroom/homebrew-cask/pull/26854, just need to re-run the build.

**Comment by bashu** [comment]
@miccal @Numbermaniac @jawshooah aaand all tests have passed

**Pull request merged**